src/ray/core_worker/reference_count.h

```diff
@@ -473,6 +473,9 @@ class ReferenceCounter : public ReferenceCounterInterface,
 
   bool IsObjectReconstructable(const ObjectID &object_id) const;
 
+  /// Relase all local references which registered on this local.
```

Comment:
Please also add a note that this should only be used on shutdown.

```suggestion
  /// Release all local references which registered on this local.
```